server/controllers/entities/lib/move_to_wikidata.js

```diff
@@ -21,7 +21,15 @@ module.exports = async (user, invEntityUri) => {
   // by the absence of the entity they just moved to Wikidata in lists generated with the help of the WQS
   await cacheEntityRelations(invEntityUri)
 
-  await mergeEntities({ userId: reqUserId, fromUri: invEntityUri, toUri: wdEntityUri })
+  await mergeEntities({
+    userId: reqUserId,
+    fromUri: invEntityUri,
+    toUri: wdEntityUri,
+    context: {
+      action: 'move-to-wikidata',
+      entityId
```

Comment:
invEntityUri === inv:${entityId}

so patches store twice the "from entity id" no ?

Reply:
yes, that could be removed

Reply:
fixed in 508e9522